drizzle/resample.py

```diff
@@ -563,9 +565,10 @@ def _increment_ctx_id(self):
 
         return plane_info
 
-    def add_image(self, data, exptime, pixmap, data2=None, scale=1.0,
+    def add_image(self, data, exptime, pixmap, data2=None,
+                  iscale=1.0, kscale=1.0,
```

Comment:
```suggestion
                  iscale=1.0, kscale=1.0,
```
Maybe `input_scale_factor` and `kernel_scale_factor`?